storagestate/storagestate.go

```diff
@@ -0,0 +1,709 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+// Copyright (C) 2022 Renesas Electronics Corporation.
+// Copyright (C) 2022 EPAM Systems, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package storagestate
+
+import (
+	"bytes"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/aoscloud/aos_common/aoserrors"
+	"github.com/aoscloud/aos_common/api/cloudprotocol"
+	"github.com/aoscloud/aos_common/utils/fs"
+	"github.com/aoscloud/aos_servicemanager/config"
+	"github.com/aoscloud/aos_servicemanager/platform"
+	"github.com/fsnotify/fsnotify"
+	log "github.com/sirupsen/logrus"
+	"golang.org/x/crypto/sha3"
+)
+
+/***********************************************************************************************************************
+ * Consts
+ **********************************************************************************************************************/
+
+const (
+	stateFile        = "state.dat"
```

Comment:
`stateFileSuffix` `_state.dat` or `stateFileFormat` `%s_state.dat`
and in the code:
```go
stateFilePath = instanceID+stateFileSuffix;
or
stateFilePath = fmt.Sprintf(stateFileFormat, instanceID)
```

Reply:
Done. Use `stateFileFormat` approach